ghga-de/ghga-service-chassis-lib
# test utils for s3

Title for squash and merge:
```
test utils for object storage
```

Description for squash and merge:
```
Added functions that serve as utils for testing linked to the 
object storage DAO:
- prepare a S3ConfigBase from an instance of a localstack test container
- populate the storage with object fixtures
- upload object to storage
- download object from storage and check its integrety
```
